src/reducers/entities/channels.ts

```diff
@@ -157,6 +157,24 @@ function channels(state: IDMappedObjects<Channel> = {}, action: GenericAction) {
             },
         };
     }
+
+    case PostTypes.RECEIVED_NEW_POST: {
+        const {channel_id, create_at} = action.data; //eslint-disable-line @typescript-eslint/camelcase
+        const channel = state[channel_id];
+
+        if (!channel) {
+            return state;
+        }
+
+        return {
+            ...state,
+            [channel_id]: {
+                ...channel,
+                last_post_at: create_at,
```

Comment:
Would there be any benefit to doing something like 
```suggestion
                last_post_at: Math.min(create_at, channel.last_post_at),
```
to ensure that this never gets out of sync if a new post event is ever received at a weird time?

Reply:
If you meant to suggest `Math.max`? yes, I think it makes it better in case of any network races.

Reply:
I remember writing `Math.min` on purpose, but I can't remember why that would've been. `Math.max` definitely sounds correct though.